bobtemplates/plone_addon/.travis.yml.bob

```diff
@@ -1,22 +1,26 @@
 language: python
+python: 2.7
 sudo: false
 cache:
-  pip: true
   directories:
-    - $HOME/buildout-cache
-python:
-  - 2.7
+  - eggs
```

Comment:
this also means that nobody plays around with his/her own settings in buildout. therefore I tend to pass in the eggs-directory too, to be explicit.